providers/js/fido-u2f-login.js

```diff
@@ -1,13 +1,15 @@
 /* global u2f, u2fL10n */
 ( function( $ ) {
-	setTimeout( function() {
-		window.console.log( 'sign', u2fL10n.request );
+	window.console.log( 'sign', u2fL10n.request );
 
-		u2f.sign( u2fL10n.request, function( data ) {
-			window.console.log( 'Authenticate callback', data );
+	u2f.sign( u2fL10n.request[0].appId, u2fL10n.request[0].challenge, u2fL10n.request, function( data ) {
+		window.console.log( 'Authenticate callback', data );
 
+		if ( data.errorCode ) {
```

Comment:
This also ensures that we don't end up with an infinite reload when U2F sign fails.